autoload/accelerated/time_driven.vim

```diff
@@ -64,8 +64,8 @@ function! accelerated#time_driven#command(cmd) "{{{
     " check timestamp
     let previous_timestamp = get(s:, a:cmd.'_timestamp_micro_seconds', [0, 0])
     let current_timestamp = reltime()
-    let [sec, microsec] = reltime(previous_timestamp, current_timestamp)
-    let msec = sec * 1000 + microsec / 1000
+    let msec = str2float(reltimestr(reltime(
```

Comment:
`str2float()` drops some digits because floating point number has fixed accuracy.  For example, when `reltimestr(reltime())` returns `"1431567707.320844"`, `str2float()` converts it to `1.431568e9` and drops some digits from the string.

I think it is better to split the string to decimal part and integer part by `"."` and to convert each part to integer.